lib/SetAPI/sampleset/SampleSetInterface.py

```diff
@@ -93,32 +79,30 @@ def create_sample_set(self, ctx, params):
                     "KBaseAssembly.SingleEndLibrary",
                     "KBaseFile.SingleEndLibrary",
                 ]
+            sample_refs = []
             for reads_ref in sample_ids:
+                # TODO: batch the ws calls instead of doing one sample at a time
                 reads_info = self.ws_client.get_object_info3(
                     {"objects": [{"ref": reads_ref}]}
-                )
-                obj_type = reads_info["infos"][0][2].split("-")[0]
+                )["infos"][0]
+                sample_refs.append(info_to_ref(reads_info))
```

Comment:
save the sample ref (although it's more n' likely that the sample_ids will already be in an appropriate form)